project/apps/account/src/app/authentication/authentication.controller.ts

```diff
@@ -23,6 +27,8 @@ export class AuthenticationController {
   @Post('register')
   public async create(@Body() dto: CreateUserDto) {
     const newUser = await this.authService.register(dto);
+    const { email, fullname } = newUser;
+    await this.notifyService.registerSubscriber({ email, fullname });
```

Comment:
await this.notifyService.registerSubscriber({ email: newUser.email, fullname: newUser.fullname });